python/metatensor_torch/metatensor/torch/atomistic/ase_calculator.py

```diff
@@ -295,7 +295,9 @@ def calculate(
         )
         calculate_energies = "energies" in properties
         calculate_forces = "forces" in properties or "stress" in properties
-        calculate_stress = "stress" in properties or "forces" in properties
+        calculate_stress = (
+            "stress" in properties or "forces" in properties
+        ) and atoms.pbc.all()
```

Comment:
```suggestion
        calculate_stress = "stress" in properties
        if calculate_stress and not atoms.pbc.all():
            warning("stress requested but likely to be wrong ………")
        
        if "forces" in properties and atoms.pbc.all():
            # we have PBC, and since the user requested forces we will run backward, so let's do stress as well
            calculate_stress = True
```